### DISCUSSION TITLE
DDC Volume doesn't work on Samsung Display M7

### DISCUSSION THREAD

**Discussion by jeansevilla** [MonitorControl/MonitorControl#948] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I can't control the volume of the monitor Samsung M7 4K UHD Smart Monitor

### Expected behavior

its a new monitor and also I use USB C to connect to the monitor, but I can't control the volume

### Environment Information (please complete the following information)

```markdown
- macOS version:
macOS Monterey 12.2.1
- Mac model:
MacBook Pro (16", 2021)
- MonitorControl version:
Versione 4.0.2 Build 6965
- Monitor(s):
Samsung M7 Monitor 
- Monitor Cable(s)/Connection(s):
MacBook Pro -> Usb type C -> Samsung M7 4K UHD Smart Monitor
- Apple Silicon/M1 (yes or no):
Yes
```

**Comment by waydabber** [comment]
Does DDC brightness control work? If so then it is probably a display firmware issue if that does not work either then it might be something else. You could try alternate apps like m1ddc or Lunar and see if those can control the volume. If not, then probably there is nothing to do about it.

**Reply by nikoskoutr** [reply]
Hello @waydabber I have just got the same monitor with the same setup (m1 macbook air connected over usb-c to the usb-c port of the monitor). DDC is not working and while observing the logs I see the following: 
`i2c error: (iokit/?) unknown subsystem error`
Most probably the display is blocking DDC over usb-c.
Have you heard about this before? I know that the API used is undocumented.
Of course I am open to any suggestions, and if I can, I can run tests and help others that have similar issues.

**Reply by waydabber** [reply]
Yes, this log means that i2c communication is blocked on that channel, usually it means the display refuses to accept commands. I had a super-expensive Lenovo mini-led display with me for a few days which did the exact same thing over USB-C but worked fine with HDMI and DisplayPort. You might want to try an alternate connection.
